source/basic.tex

```diff
@@ -1336,13 +1332,8 @@
 \end{example}
 
 \pnum
-A name used in the definition of a class \tcode{X} outside of a member
-function body, default argument, \grammarterm{noexcept-specifier},
-default member initializer\iref{class.mem},
-contract condition\iref{dcl.attr.contract},
-or nested class definition\footnote{This refers to unqualified names
-following the class name; such a name may be used in the
-\grammarterm{base-clause} or may be used in the class definition.}
```

Comment:
Is the removal of this footnote intentional? Clarifying that this covers the base-specifiers seems at least moderately useful.

Reply:
It seemed so... obvious to me, but we seem to have several similar footnotes in the vicinity. Restored.